### ISSUE TITLE
completion is not compatible with zsh

### ISSUE THREAD

**Issue by ssbarnea** [shanesmith/gerrit-cli#23]
If your shell is `zsh`, you cannot use `eval "$(gerrit completion)"` because completion command does not exists.

**Comment by shanesmith** [comment]
Yup that's true, it is currently only written for bash. It's a pretty simple script though so I'll take a shot at converting it to zsh... unless you happen to know all about zsh completion and would like to contribute? =)
